test/Microsoft.AspNetCore.Server.Kestrel.Performance/Requests.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Linq;
+using System.Text;
+using Microsoft.AspNetCore.Server.Kestrel.Internal;
+using Microsoft.AspNetCore.Server.Kestrel.Internal.Http;
+using Microsoft.AspNetCore.Server.Kestrel.Internal.Infrastructure;
+using Microsoft.AspNetCore.Testing;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Performance
+{
+    public class Requests
+    {
+        public const int Pipelining = 16;
+
+        private const string plaintextRequest = "GET /plaintext HTTP/1.1\r\nHost: www.example.com\r\n\r\n";
+
+        private const string liveaspnetRequest = "GET https://live.asp.net/ HTTP/1.1\r\n" +
```

Comment:
Just deleted the `.` from the domain name; going on the important words being `Site`; `Request` and `Pipelined` wasn't thinking to hard about it :)